src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -161,6 +161,32 @@
  * }
  * </pre>
  * <p>
+ * To configure the check with policy {@code alone} for
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_SWITCH">
+ * LITERAL_SWITCH</a> tokens:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;RightCurly&quot;&gt;
+ *  &lt;property name=&quot;option&quot; value=&quot;alone&quot;/&gt;
+ *  &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_SWITCH&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <pre>
+ * class Test {
+ *
+ *     public void method0() {
+ *         int mode = 0;
+ *         switch (mode) {
+ *             case 1:
+ *                 int x = 1;
+ *                 break;
+ *             default:
+ *                 x = 0; } // violation '}' at column 24 should be alone on a line'
```

Comment:
Violation case is ok, we need another example in this same test class and example of good code with no violation.

Reply:
done